charts/kubernetes-graphql-gateway/values.yaml

```diff
@@ -0,0 +1,77 @@
+cors:
+  enabled: false
+  allowedHeaders: "*"
+  allowedOrigins: "*"
+
+image:
+  name: ghcr.io/openmfp/kubernetes-graphql-gateway
+  pullPolicy: IfNotPresent
+
+kcp:
+  enabled: true
+
+gateway:
+  port: 8080
+  healthCheck:
+    enabled: true
+    port: 3389
+  metricsPort: 8081
+  graphiql: true
+  logLevel: trace
+  shouldImpersonate: true
+  usernameClaim: email
+
+listener:
+  enabled: true
+  port: 8090
+  healthCheck:
+    enabled: true
+    port: 3390
+  metricsPort: 8091
+
+imagePullSecret: ghcr-credentials
+
+deployment:
+  maxUnavailable: 0
+  maxSurge: 5
+
+
+sentry:
+  environment: dev
+
+istio:
+  enabled: true
+  gateway:
+    name: gateway
+
+virtualService:
+  pathPrefix: /kubernetes-graphql-gateway/
+  hosts:
+    - "*"
+  httpRules:
+    - name: default
+      cors:
+        allowHeaders:
+          - "*"
+        allowMethods:
+          - GET
+          - POST
+        allowOrigins:
+          - regex: .*
+
+resources:
```

Comment:
are those in use? Lets make it possible to configure different resource requests for listener and gateway

Reply:
Now those values are defined separately for the listener and gateway and added to deployment template only in case it is defined.

Reply:
Tested - values if defined are present in the deployment instance.